core/qthelper.cpp

```diff
@@ -1065,7 +1065,7 @@ static void convertLocalFilename(const QHash<QString, QByteArray> &hashOf, const
 		QByteArray hash = it.key();
 		HashToFile dummy { hash, QString() };
 		for(auto it2 = std::lower_bound(h2f.begin(), h2f.end(), dummy);
-		    it2 != h2f.end() && it2->hash == hash; ++it2) {
+			it2 != h2f.end() && it2->hash == hash; ++it2) {
```

Comment:
no that's wrong. This is a continuation line of the `for` statement and therefore aligns with `auto` using spaces

Reply:
correct, sorry I have corrected it.

Actually it is not a continuation of the for statement since that ends with a ;

Reply:
I'm sorry to be a pain here, but...
```
		for(auto it2 = std::lower_bound(h2f.begin(), h2f.end(), dummy);
		    it2 != h2f.end() && it2->hash == hash; ++it2) {
```
this is one for statement. `for (init; end_condition; modifier) {`
where 
`init` is `auto it2 = std::lower_bound(h2f.begin(), h2f.end(), dummy)`
`end_condition` is `it2 != h2f.end() && it2->hash == hash`
`modifier` is `++it2`

So the `;` is the internal separator inside the arguments of the `for` keyword.

Reply:
you are not a pain, you are actually making me feel embarrassed since I did not catch that :-(